pages/queryKeys.ts

```diff
@@ -1,5 +1,7 @@
 const queryKeys = {
   COMMENT: "comment",
+  USER: "user",
```

Comment:
These names are not descriptive enough actually. Something like 'GET_COMMENTS', 'GET_COMMENT', 'ADD_USER'